pages/api/openPR.js

```diff
@@ -15,40 +15,199 @@ const GITHUB_BRANCH = process.env.GITHUB_BRANCH
+    name = values.organizations[i].name;
+    website = values.organizations[i].website;
+    org_type = values.organizations[i].org_type;
+
+    values.organizations[i] = {
+      name: name,
+      website: website,
+      org_type: org_type,
+    };
+  }
+  return values;
+}
+
+// Return multiple submission files including corresponding filepaths
+function getSubmissionFiles(values, myJSON) {
+  let name,
+    nomineePath,
+    dpgPath,
+    stage,
+    data,
+    files = {};
+
+  stage = values.stage;
+
+  // Delete multiple DPG nomination fields
+  [
+    "aliases",
+    "description",
+    "website",
+    "type",
+    "SDGs",
+    "license",
+    "organizations",
+    "stage",
+    "repositoryURL",
+  ].forEach((e) => delete values[e]);
+
+  // Convert JavaScript submission sorted object into JSON string and add newline at EOF
+  data = JSON.stringify(values, null, 2).concat("\n");
+
+  // Parse project names using DPG naming convention for filenames
+  name = parseProjectName(values);
+
+  // Add nominee submission to nominee directory
+  nomineePath = "nominees/" + `${name}`;
+
+  // Add DPG submission to screening directory
+  dpgPath = "screening/" + `${name}`;
+
+  stage === "nominee"
+    ? Object.entries(myJSON).forEach(
+        ([key, value]) =>
+          (files = {
+            [nomineePath]: {
+              content: myJSON,
+              encoding: "utf-8",
+            },
+          })
+      )
+    : Object.entries(myJSON).forEach(
+        ([key, value]) =>
+          (files = {
+            [nomineePath]: {
+              content: myJSON,
+              encoding: "utf-8",
+            },
+            [dpgPath]: {
+              content: data,
+              encoding: "utf-8",
+            },
+          })
+      );
+
+  return files;
+}
+
+// Nominee processing before opening pull request
+function nomineeSubmission(values, sortedSubmission) {
+  // Sort entries by iterating through object and unpacking entries
+  Object.entries(values).forEach(
+    ([key, value]) =>
+      (sortedSubmission = {
+        name: values.name ? values.name : "",
+        aliases: values.aliases ? [values.aliases] : [""],
+        description: values.description ? values.description : "",
+        website: values.website ? values.website : "",
+        license: values.license ? values.license : [],
+        SDGs: values.SDGs ? values.SDGs : [],
+        sectors: values.sectors ? values.sectors : [],
+        type: values.type ? values.type : [],
+        repositoryURL: values.repositoryURL ? values.repositoryURL : "",
+        organizations: values.organizations ? [values.organizations] : [],
+        stage: values.stage ? values.stage : "",
+      })
+  );
+
+  // Order nominee fields in the correct order e.g organizations
+  sortedSubmission = orderFields(sortedSubmission);
+  return sortedSubmission;
+}
+
+// DPG candidate processing before opening pull request
+function dpgSubmission(values, sortedSubmission) {
```

Comment:
How is this function different from `nomineeSubmission` ?

Reply:
Resolved in commit 6151335.